.pre-commit-config.yaml

```diff
@@ -18,6 +18,13 @@ repos:
     args: [--git-add]
     pass_filenames: false
     files: CONTRIBUTORS.md
+  - id: manpage
+    name: Update manpages from markdown source files
```

Comment:
This means that `pandoc` is now a requirement for development.  Probably we should mention this in HACKING.md?

Reply:
Also, should we add this same pandoc invocation to the PR Github actions?

Reply:
`pandoc` is only a requirement if you change `PERMISSIONS.md`. Like `inkscape` is a requirement if you change `socranop/data/xdg/io.github.socratools.socranop.svg`.

You are right, this should be in the GitHub PR actions. Then the PR at the latest will catch if someone without pandoc has modified `PERMISSIONS.md` and committed `PERMISSIONS.md` without updating `socranop/data/man/socranop-permissions.7`.

Reply:
Hmm. There is a Python package called pandoc containing a python library called `pandoc`. This python library appears to not require the system `pandoc` utility`. Let me examine whether the Python package could replace the system-wide `pandoc` utility for our purposes.

Reply:
These two should now be addressed:

> This means that `pandoc` is now a requirement for development. Probably we should mention this in HACKING.md?

> Also, should we add this same pandoc invocation to the PR Github actions?

Reply:
Having the pandoc utility generate the man page on github creates a new problem: The github VM runs pandoc-2.5, and my Fedora system uses pandoc-2.9.something. They produce very different output:

```patch
 @@ -1,6 +1,6 @@
-.\" Automatically generated by Pandoc 2.9.2.1
+.\" Automatically generated by Pandoc 2.5
 .\"
-.TH "SOCRANOP-PERMISSIONS" "7" "2021-07-18" "${PACKAGE} ${VERSION}" "Miscellaneous"
+.TH "SOCRANOP\-PERMISSIONS" "7" "2021\-07\-18" "${PACKAGE} ${VERSION}" "Miscellaneous"
 .hy
```

So I need to either
  * tell the pandoc utility to always use the same style of hyphen (always `\-`) regardless of the pandoc version
  * try the pandoc Python package and hope it does better

Having `sed` delete the `Automatically generated` line is trivial.

Reply:
I have separated the pandoc generated socranop-permissions(7) man page to https://github.com/socratools/socranop/pull/55.